### DISCUSSION TITLE
print not possible once p:media is loaded

### DISCUSSION THREAD

**Discussion by Rachana010693** [primefaces/community#2582] [PrimeFaces] [answered]
I'm encountering an issue with the PDF viewer in our legacy application. While the PDF loads successfully in the p media component but, there's a problem with the button located below it(button is since a long legacy app). When this button is clicked, a simple JavaScript function (window.print()) is supposed to trigger the print preview dialog. However, the print preview dialog does not open.

I've observed similar behavior on the showcase site: it doesn't work in Chrome, but it functions correctly in Firefox.

On chrome browser not working:
<img width="944" alt="chromess" src="https://github.com/user-attachments/assets/70153a37-f700-4f12-a3f5-1d20ad9ca1f7">

Firefox:
<img width="956" alt="image" src="https://github.com/user-attachments/assets/9fdc45e5-c12c-45c1-ac6a-9067fb451540">

Can someone please help me on this ? any leads will be very much appreciated.

**Comment by melloware** [comment, marked as answer]
Feels like a Chrome bug or issue.
